connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -2120,7 +2130,6 @@ class SparkConnectPlanner(val session: SparkSession) {
 
       case StreamingQueryCommand.CommandTypeCase.EXPLAIN =>
         val result = query match {
-          case q: StreamExecution => q.explainInternal(command.getExplain.getExtended)
```

Comment:
Sorry double checked and found that the query was returned using `StreamingQueryManager`'s `get` method, which is StreamingQuery https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryManager.scala#L106

Following existing code maybe we could change this to 
`case q: StreamingQuery` and do a cast like:
https://github.com/apache/spark/blob/35503a535771d257b517e7ddf2adfaefefd97dad/sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamSuite.scala#L249